### PR TITLE
Implemented predict prodigy functionality.

### PR DESCRIPTION
**Pull request by Eleni170**

### CODE REVIEW

File: mantis/cli.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import typer
+
+from mantis.predict import predict_prodigy as predict_prodigy
```
Comment (nsorros) [inline mantis/cli.py:3]: is the `as` needed? isn't this enough `from mantis.predict import predict_prodigy`

File: mantis/predict.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        for line in f:
+            pattern = json.loads(line)
+            text = pattern["text"]
+            meta = pattern["meta"]
+            true.append(meta)
+            doc = spacy_model(text)
+            pred.append(create_prodigy_spans(doc))
+
+    tags_list = tags.split(",")
+    evaluator = Evaluator(true=true, pred=pred, tags=tags_list, loader=None)
```
Comment (nsorros) [inline mantis/predict.py:44]: I would not do the following lines. I would save the predictions that Evaluator accepts and then the user would need to run
`nervaluate true_path pred_path`. So this function role is just to create the `pred_path` data

**Review by nsorros — CHANGES_REQUESTED** [review]
Just a small API change and this is ready to go

**Comment by Eleni170** [comment]
Ok I understood what you are talking about. You can see the changes.

File: tests/test_predict.py [hunk 3]
```diff
@@ -9,64 +9,4 @@ def predict_prodigy(tmp_path):
...
-            "recall": 1.0,
-            "f1": 1.0,
-        },
-    }
-
-    assert results["strict"] == expected["strict"]
-    assert results["ent_type"] == expected["ent_type"]
-    assert results["partial"] == expected["partial"]
-    assert results["exact"] == expected["exact"]
+    predict_prodigy(model_path=model_path, data_path=data_path)
```
Comment (nsorros) [inline tests/test_predict.py:12]: I guess you can assert that data are created

**Review by nsorros — APPROVED** [review]

**Pull request merged**